test/integration/src/main/java/io/pravega/test/integration/selftest/TestConfig.java

```diff
@@ -34,14 +34,14 @@
     static final Property<Integer> CONTAINER_COUNT = Property.named("containerCount", 1);
     static final Property<Integer> STREAM_COUNT = Property.named("streamCount", 100);
     static final Property<Integer> SEGMENTS_PER_STREAM = Property.named("segmentsPerStream", 1);
-    static final Property<Integer> TRANSACTION_FREQUENCY = Property.named("transactionFrequency", 100);
-    static final Property<Integer> MAX_TRANSACTION_SIZE = Property.named("maxTransactionSize", 10);
+    static final Property<Integer> TRANSACTION_FREQUENCY = Property.named("transactionFrequency", Integer.MAX_VALUE);
```

Comment:
Is this change desired? I don't understand how transactionFrequency relates.

Reply:
This is the default configuration of the SelfTester. Any of these can be overridden from the command line when running a test. I simply elected to have transactions off by default for perf tests so they don't interfere with perceived performance. So yes, this is harmless.